src/applications/appeals/996/pages/addIssue.js

```diff
@@ -1,4 +1,4 @@
-import { SELECTED, MAX_LENGTH } from '../constants';
+import { SELECTED, MAX_LENGTH } from '../../shared/constants';
```

Comment:
This `MAX_LENGTH` is using:
- `ISSUE_NAME` not in the shared folder, and
- `SELECTIONS` which is in the shared folder
So we're going to have to use that pattern where we define: `import { MAX_LENGTH as HLR_MAX_LENGTH } from '../constants';` that we discussed before

Reply:
What's in shared is the one for HLR right?